examples/07_joining_tables_with_FuzzyJoin.py

```diff
@@ -0,0 +1,161 @@
+"""
+Joining tables with fuzzyjoin
+================================
+
+In this example, we show how to join tables with the :function:`fuzzyjoin` function. 
+We also demonstrate why this method is the most easy and appropriate tool for handling 
+the joining of tables for users that want to improve their machine learning models quickly.
+We go through some of the rich options that allow for manipulation when joining tables,
+such as taking into account the precision of the match.
+
+To do so, we will predict the happiness score of a country from
+the 2022 [World Happiness Report](https://worldhappiness.report/).
```

Comment:
This is not the correct syntax for restructured-text links (check the rendered doc)